keras_cv/models/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.models.densenet import DenseNet
```

Comment:
do we expect end user to directly use DenseNet?

Reply:
Good question, I see no issue with it in the new API, but maybe @fchollet has excluded these base models on purpose in the past?

Reply:
Just wanted to understand. 

Will we be porting all the models from `keras.applications` to `keras_cv.models`?

Reply:
Just wanted to understand. 

Will we be porting all the models from `keras.applications` to `keras_cv.models`?

Reply:
Probably not.  Some of them are pretty outdated and probably not worth the effort I.e. VGG

Reply:
We don't expect users to instantiate their own custom DenseNet and we won't be able to provide weights for all possible variations, so we should not export it. Users who want a custom DenseNet can simply copy the code (which is very modular and reusable).